### ISSUE TITLE
[HIPIFY] CUDA_VERSION should not be hipified

### ISSUE THREAD

**Issue by mangupta** [ROCm-Developer-Tools/HIPIFY#372]
PR #365 added a mapping from CUDA_VERSION to HIP_VERSION. This is incorrect. Doing this implies there is a 1 to 1 mapping between CUDA_VERSION and HIP_VERSION. So if I write some code that guards against a feature introduced in a specific version of CUDA, such as below:
```c
#if defined(CUDA_VERSION) && CUDA_VERSION < 9000
// Use fallback for older CUDA versions
#else
// Use feature introduced in CUDA 9.x. Cooperative launch for example
#endif
```
The above code would not use the right code paath since there is no 1 to 1 mapping between HIP_VERSION and CUDA_VERSION. Instead it might be better to insert a comment such as ```//TODO_HIPIFY: Developer needs to port this manually``` or something of that sort. At the least, leaving it un-hipified is probably better.

**Comment by emankov** [comment]
> The above code would not use the right code path since there is no 1 to 1 mapping between HIP_VERSION and CUDA_VERSION.

Could you provide the correct mapping between `HIP_VERSION` and `CUDA_VERSION`?

**Comment by mangupta** [comment]
> 
> Could you provide the correct mapping between `HIP_VERSION` and `CUDA_VERSION`?

I am not sure that I understood your question. As I mentioned above that we cannot map CUDA_VERSION to HIP_VERSION, since HIP versions don't track CUDA versions. There are some features in each CUDA release that we don't support in HIP. Even if support for them is added in HIP, it is not added in the same chronological order as it was added to in CUDA. Hence we should not map CUDA_VERSION to HIP_VERSION.

**Comment by emankov** [comment]
Thanks, Maneesh for your explanations. Fixed with #373.
